service-base/src/main/java/org/eclipse/hono/service/http/HttpContext.java

```diff
@@ -73,4 +89,137 @@ public void setTracingContext(final SpanContext spanContext) {
     public SpanContext getTracingContext() {
         return spanContext;
     }
+
+    @Override
+    public QoS getRequestedQos() {
```

Comment:
Based on today's Community call: @ctron mentioned that this will cause problem with Quarkus. I see your point (I guess: "Design for inheritance, or prohibit it") but wasn't sure anymore after the discussion.

Reply:
Let's wait and see what happens with the Quarkus story ... in the meantime I suggest to follow good design practice here :-)